CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jasonnutter @tnorling @sameerag @pkanher617 @DarylThayil @jo-arroyo @jmckennon @technical-boy
```

Comment:
I am curious to check specifying paths here and how that works.

Reply:
Example:

```
# Request reviews from Sameera and Santiago when changes are made to msal-node
/lib/msal-node @sameerag @sangonzal 
```